nilearn/_utils/cache_mixin.py

```diff
@@ -108,11 +108,11 @@ def cache(func, memory, func_memory_level=None, memory_level=None,
     memory: instance of joblib.Memory or string
         Used to cache the function call.
 
-    func_memory_level: int, optional
+    func_memory_level: int
```

Comment:
Sorry I wasn't clear what I meant is that you didn't do what you announced in the description of the PR.

> In CacheMixin, make memory levels mandatory for function _cache. It is logic and they are never used without them.

Both `func_memory_level` and `memory_level` are still optional parameters atm ..